examples/membrane/memv.in

```diff
@@ -22,10 +23,14 @@ sdens 10.0
 temp 298.15
 calcenergy comps
 calcforce no
-zmem -10
-lmem 20
-mdie 2.0
-memv -2.0
+# WARN: zmem is no longer supported
+# zmem -10
+# WARN: lmem is no longer supported
+# lmem 20
+# WARN: mdie is no longer supported
+# mdie 2.0
+# WARN: memv is no longer supported
+# memv -2.0
```

Comment:
The `zmem`, `lmem`, `mdie`, and `memv` are keywords that cannot be found in any of the input file specifications or anywhere on the readthedocs site.

Reply:
I think we should delete this example.

Reply:
Done.